### ISSUE TITLE
Some problems when reproducing

### ISSUE THREAD

**Issue by WoshiBoluo** [ArmanAfrasiyabi/MixtFSL-fs#2]
Hello, I am very interested in your research and reproduced it with miniImageNet on RN12. 
My settings are 5way-1shot, T=0.025, m=0.01, r=0.95, Te=0.2, me=0.001 , The results obtained are 62.48, 78.96, which cannot reach the results in the paper. Is there a problem with my settings? 
Can you give the correct setting for the paper result(like choosing 5 or 1shot when training, how to choose the values of T and m)? 
In addition, I don’t understand why the Te and me settings are different during training and testing, what impact will this difference have?
Looking forward to your reply.

**Comment by ArmanAfrasiyabi** [comment]
Hello,

Thank you for your interest. Please use 1-shot and 5-shot for episodic validation to reproduce 1-shot and 5-shot, respectively. For example, early-stoping with 1-shot while reproducing 5-shot results could be one possible reason to end up with lower numbers. Another possible issue could be using different PyTorch setups. Please follow our guidelines for PyTorch and Torchvision. Specifically, I am using the following settings: Python 3.8.10, PyTorch 1.9.1+cu111, CUDA Version: 11.4 installed on Ubuntu 20.04.3 LTS. The miniImageNet results are produced by NVIDIA GeForce RTX 2080 Ti graphic card.     

As discussed in the supplementary materials of the work, we used the validation set to tune the temperature hyperparameter. This is why the Te is different during the training and testing setups. Additionally, the general setting for query size could vary since our work stays in an inductive framework, and the query size would not have an impact. However, we recommend keeping the default query size during training.
